saleor/graphql/order/mutations/order_lines_create.py

```diff
@@ -197,6 +208,24 @@ def perform_mutation(  # type: ignore[override]
 
         return OrderLinesCreate(order=order, order_lines=added_lines)
 
+    @classmethod
+    def get_variant_rule_info_map(cls, data, channel_id, language_code):
```

Comment:
@IKarbowiak how about adding one benchmark for this function to confirm that we don't perform multiple database queries here?

Reply:
Done